src/app/ApplicationTemplate.Shared/Presentation/Welcome/OnboardingPageViewModel.cs

```diff
@@ -3,23 +3,33 @@
 using Chinook.DynamicMvvm;
 using Chinook.SectionsNavigation;
 using Chinook.StackNavigation;
+using Microsoft.Extensions.Localization;
 
 namespace ApplicationTemplate.Presentation
 {
 	public partial class OnboardingPageViewModel : ViewModel
 	{
+		public OnboardingPageViewModel()
+		{
+			OnboardingItems = new OnboardingItemViewModel[]
+			{
+				new OnboardingItemViewModel(this.GetService<IStringLocalizer>()["Onboarding_Content"], "ms-appx:///Assets/Tutorial_FirstScreen_Icon.png"),
+				new OnboardingItemViewModel(this.GetService<IStringLocalizer>()["Onboarding_Content"], "ms-appx:///Assets/Tutorial_SecondScreen_Icon.png"),
+				new OnboardingItemViewModel(this.GetService<IStringLocalizer>()["Onboarding_Content"], "ms-appx:///Assets/Tutorial_ThirdScreen_Icon.png"),
+			};
+		}
+
 		public IDynamicCommand NavigateToLogin => this.GetCommandFromTask(async ct =>
 		{
 			await this.GetService<ISectionsNavigator>().NavigateAndClear(ct, () => new LoginPageViewModel(isFirstLogin: true));
 			await this.GetService<IApplicationSettingsService>().CompleteOnboarding(ct);
 		});
 
-		public OnboardingItemViewModel[] OnboardingItems { get; } = new[]
+		public OnboardingItemViewModel[] OnboardingItems
 		{
-			new OnboardingItemViewModel("Get your shot of good humor everyday. Read the latest and greatest dad jokes!", "ms-appx:///Assets/Tutorial_FirstScreen_Icon.png"),
-			new OnboardingItemViewModel("Get your shot of good humor everyday. Read the latest and greatest dad jokes!", "ms-appx:///Assets/Tutorial_SecondScreen_Icon.png"),
-			new OnboardingItemViewModel("Get your shot of good humor everyday. Read the latest and greatest dad jokes!", "ms-appx:///Assets/Tutorial_ThirdScreen_Icon.png"),
-		};
+			get => this.Get<OnboardingItemViewModel[]>();
```

Comment:
This never changes so you don't need a dynamic property. Use a get-only normal property instead.

Reply:
If OnboardingItems is not a dynamic property, I cannot use IStringLocalizer to set the content, because it gives the error "Cannot reference a non-static field". I then have to use ResourceLoader. So is the use of IStringLocalizer or a get-only property the best?